lib/src/default_extension_map.dart

```diff
@@ -434,6 +434,7 @@ const Map<String, String> defaultExtensionMap = <String, String>{
   'mc1': 'application/vnd.medcalcdata',
   'mcd': 'application/vnd.mcd',
   'mcurl': 'text/vnd.curl.mcurl',
+  'md': 'text/markdown',
```

Comment:
```suggestion
  // https://www.rfc-editor.org/rfc/rfc7763
  'md': 'text/markdown',
```

Going forward, I think it'd be good to include these in all additions. Keeps everyone honest!